src/CSSList/CSSList.php

```diff
@@ -281,7 +281,7 @@ public function remove($oItemToRemove)
     /**
      * Replaces an item from the CSS list.
      *
-     * @param RuleSet|Import|Charset|CSSList $oItemToRemove
+     * @param RuleSet|Import|Charset|CSSList $oOldItem
      *        May be a RuleSet (most likely a DeclarationBlock), a Import, a Charset
      *        or another CSSList (most likely a MediaQuery)
```

Comment:
Shouldn't this also add a `@param` for `$mNewItem?`

Reply:
Sort of :-) … in this PR, I would like to mostly update existing PHPDoc comments. As there are lots of type annotations missing, I plan to add them in several dedicated PRs.